src/hermes/cli.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module provides the main entry point for the HERMES command line application.
+"""
+import typing as t
+
+import click
+
+from hermes.commands import workflow
+
+
+class WorkflowCommand(click.Group):
+    """
+    Custom multi-command that implements
+
+    - sub-commands in a fixed order,
+    - automatic call of sub-commands, and
+    - filtering of commands to be executed.
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        self._command_order = []
+        super().__init__(*args, **kwargs)
+
+    def add_command(self, cmd: click.Command, name: t.Optional[str] = None) -> None:
+        """
+        Overwritten to ensure the order of commands is retained.
```

Comment:
```suggestion
        Overridden to ensure the order of commands is retained.
```

Think this is an override rather than a replacement?